backend/dataall/base/aws/kms.py

```diff
@@ -69,3 +69,17 @@ def check_key_exists(self, key_alias: str):
             return None
         else:
             return key_exist
+
```

Comment:
This function is not used at the moment. @noah-paige should we remove it and move the `kms` handler back to the data_sharing module?

Reply:
I moved it back to avoid issues with the work done in #284